apps/web/pages/api/integrations/alby/webhook.ts

```diff
@@ -0,0 +1 @@
+export { default, config } from "@calcom/features/ee/payments/api/alby-webhook";
```

Comment:
Since it's just an import from there, could we just move the file in here directly. That way, we don't keep alby related code in core.

Reply:
this was fixed by Alan